### PR TITLE
resourcewatch: copy output into resourcewatch.log

### PR DESCRIPTION
**Pull request by mfojtik**
Preserve the resource watch logs in artifacts.

### CODE REVIEW

**Review by petr-muller — APPROVED** [review]

**Review by petr-muller — APPROVED** [review]

**Review by petr-muller — APPROVED** [review]

File: pkg/steps/clusterinstall/template.go [hunk 1]
```diff
@@ -148,11 +148,18 @@ objects:
 
         trap 'jobs -p | xargs -r kill || true; exit 0' TERM
 
-        runwatcher &
+        runwatcher &> /tmp/artifacts/resourcewatch.log &
```
Comment (wking) [inline pkg/steps/clusterinstall/template.go:151]: Shouldn't this have been [`${ARTIFACT_DIR}/resourcewatch.log`][1]?  And don't we already preserve these logs, like [this][2]?

[1]: https://steps.svc.ci.openshift.org/help#env
[2]: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-etcd-operator/427/pull-ci-openshift-cluster-etcd-operator-master-e2e-azure/1297881221895819264/artifacts/e2e-azure/gather-extra/container-logs/

**Pull request merged**